scss/misc/_search-input.scss

```diff
@@ -29,3 +29,7 @@ input.search-input {
     width: calc(100% - 80px);
   }
 }
+
+input#chat-list-search {
+  color: #fff;
```

Comment:
hardcoded white? can you use a theme variable?

Reply:
Didn't use a themevariable as currently also the X icon is a hardcoded white. But for sure, a theming variable makes sense here. But I have to admit that I hate adding theming variables :D

Reply:
yeah, but I believe the var you are searching already exists: `navBarText`